### ISSUE TITLE
Steamwheedle Cartel reputation quests not ingame

### ISSUE THREAD

**Issue by Sylvestersnr** [elysium-project/server#3016]
The following quests are not in-game, but their NPC's exist, this is a stock mangos problem, with the actual quests not having been added.

[Rachet](https://wow.gamepedia.com/Quest:Mending_Old_Wounds)
[Booty Bay](https://wow.gamepedia.com/Quest:Traitor_to_the_Bloodsail)
[Gadgetzan](https://wow.gamepedia.com/Quest:War_at_Sea)
[Everlook](https://wow.gamepedia.com/Quest:Making_Amends)

**Comment by Nelythia** [comment]
These quests can only be done when you are below neutral at the specific reputation.
If you are above neutral you can not see nor do them.

**Comment by QAkvastskaft** [comment]
These quests are only available after patch 1.11 and below neutral in reputation.
However quest 9266 is completely missing.

From the 1.11 patch notes:
"New repeatable "turn-in" quests have been added to each of the goblin cities as a vehicle to raise the player's faction with them (while decreasing the player's Bloodsail Buccaneer faction)"

Each goblin city being:
Rachet
Booty Bay
Gadgetzan
Everlook

Also here NPC id and quest ID is consistent with patch 1.11 as seen here:
https://web.archive.org/web/20071118220955/http://wow.allakhazam.com:80/db/mob.html?wmob=16416

**Comment by Sylvestersnr** [comment]
You're right, I switched out the ''stock mangos problem'', what I meant was that, it was a stock mangos problem that the Everlook quest from Bronn, is not in the game.
